### DISCUSSION TITLE
Welcome to jwt-scala Discussions!

### DISCUSSION THREAD

**Discussion by erwan** [jwt-scala/jwt-scala#179] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by reiniersf** [comment]
Hello, is the jwt-spray-json support limited to Scala 2.13 only? Is there any support for Scala 3?

**Reply by erwan** [reply]
Hi,

Yes, last time I tried I had some issues with Spray Json and Scala 3 so considering it's no longer maintained I just gave up.

The best is to switch to a different library, but if you want really Scala 3 support for jwt-spray-json feel free to make a PR!
